src/cancelContentPlay.js

```diff
@@ -6,11 +8,19 @@ then signalling that we should play after the ad is done.
 */
 
 export default function cancelContentPlay(player) {
-  if (player.ads.cancelPlayTimeout) {
+  if (pm.isMiddlewareMediatorSupported()) {
+    // Don't use cancelContentPlay while playMiddleware is in use
+    return;
+  } else if (player.ads._shouldBlockPlay === false) {
+    // Don't block play if explictly set to not block
```

Comment:
To make this comment more specific, maybe "Only block play if the ad plugin is in a state where play should be blocked. This currently means BeforePreroll and Preroll."